core/lib/src/lib.rs

```diff
@@ -126,6 +126,7 @@ pub mod http {
     pub use rocket_http::*;
 }
 
+#[macro_use] mod unix;
```

Comment:
Where do we use the macros from `macro_use`? And, can we use `unix::foo!()` instead?

Reply:
Not sure why, but I got this error when not using macro_use:

![image](https://user-images.githubusercontent.com/1556054/60762938-5a30ca00-a09c-11e9-85f9-1fbb87d3e93c.png)

Reply:
My understanding is that macro_rules macros still don't exist in the defining module - they exist in the crate root, unless re-exported with a `use as`. They also must still be defined before use, and annotated with a `#[macro_use]`.